app/views/search/_result_geo.html.erb

```diff
@@ -8,9 +8,7 @@
   </div>
 
   <% if result_geo['contributors'] %> 
-    <p class="authors truncate-list">
```

Comment:
What happens to the `truncate-list` class in this partial? Are we not using it anymore?

Reply:
Ohh, good catch. We *should* still be using it, but I was writing this code during a conference and not paying very much attention. 🙂 Change forthcoming...